projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,660 @@
+- name: "Image Laterality"
+  group: 0x0020
+  element: 0x0062
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Frame of Reference UID"
+  group: 0x0020
+  element: 0x0052
+  op: "replace_UID"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Synchronization Frame of Reference UID"
+  group: 0x0020
+  element: 0x0200
+  op: "replace_UID"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Position Reference Indicator"
+  group: 0x0020
+  element: 0x1040
+  op: "replace"
+#CT, MR, PET, US, X-Ray, and RT Images
+#################################### 0028 Group ###################################
+#
+#
+- name: "Samples Per Pixel"
+  group: 0x0028
+  element: 0x0002
+  op: "keep"
+#https://dicom.innolitics.com/ciods/ct-image/image-pixel/00280002
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00280002
+#https://dicom.innolitics.com/ciods/mr-image/image-pixel/00280002
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00280002
+#https://dicom.innolitics.com/ciods/ultrasound-image/image-pixel/00280002
+#https://dicom.innolitics.com/ciods/ultrasound-image/us-image/00280002
+#https://dicom.innolitics.com/ciods/rt-image/image-pixel/00280002
+#https://dicom.innolitics.com/ciods/rt-image/rt-image/00280002
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Photometric Interpretation"
+  group: 0x0028
+  element: 0x0004
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Planar Configuration"
+  group: 0x0028
+  element: 0x0006
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Rows"
+  group: 0x0028
+  element: 0x0010
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Columns"
+  group: 0x0028
+  element: 0x0011
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Pixel Spacing"
+  group: 0x0028
+  element: 0x0030
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Bits Allocated"
+  group: 0x0028
+  element: 0x0100
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Bits Stored"
+  group: 0x0028
+  element: 0x0101
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "High Bit"
+  group: 0x0028
+  element: 0x0102
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Pixel Representation"
+  group: 0x0028
+  element: 0x0103
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Burned In Annotation"
+  group: 0x0028
+  element: 0x0301
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Recognizable Visual Features"
+  group: 0x0028
+  element: 0x0302
+  op: "keep"
+#CT, MR, PET, US, X-Ray, and RT Images
+- name: "Pixel Spacing Calibration Type"
+  group: 0x0028
+  element: 0x0a02
+  op: "keep"
+#X-Ray
+- name: "Pixel Spacing Calibration Description"
+  group: 0x0028
+  element: 0x0a04
+  op: "keep"
+#X-Ray
+- name: "Pixel Intensity Relationship"
+  group: 0x0028
+  element: 0x1040
+  op: "keep"
+#X-Ray
+- name: "Pixel Intensity Relationship Sign"
+  group: 0x0028
+  element: 0x1041
+  op: "keep"
```

Comment:
I would say worth starting a file for these and moving the corresponding tags that aren't in all CT, MR, US, RT, PET and X-ray into their specific files:
- xr (x-ray)
- ct
- rt